logs_processors.go

```diff
@@ -93,6 +94,14 @@ type GrokRule struct {
 	MatchRules   *string `json:"match_rules"`
 }
 
+// LookupProcessor represents the lookup processor from config API.
+type LookupProcessor struct {
+	Source        *string  `json:"source"`
```

Comment:
The fields that are not required should be marked as `omitempty` (IIUC that's `default_lookup`)

Reply:
You're right! I have made the change 👌